ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/README.md

```diff
@@ -1,17 +1,33 @@
 # Analyze hotspots
 
-This sample demonstrates how to execute a geoprocessing task to calculate a hotspot analysis based on the frequency of 911 calls. It calculates the frequency of these calls within a given study area during a specified constrained time period set between 1/1/1998 and 5/31/1998.
+Use a geoprocessing service and a set of features to identify statistically significant hot spots and cold spots.
 
 ![](screenshot.png)
 
+## Use case
+
+This tool identifies statistically significant spatial clusters of high values (hot spots) and low values (cold spots). For example, a hotspot analysis based on the frequency of 911 calls within a set region.
+
+## How to use the sample
+
+Select a date range (between 1998-01-01 and 1998-05-31) from the dialog and tap on Analyze. The results will be shown on the map upon successful completion of the `GeoprocessingJob`.
+
 ## How it works
-A `GeoprocessingTask` is created by setting the URL to the REST endpoint of a geoprocessing service. `GeoprocessingParameters` are created and the execution type is set to `AsynchronousSubmit`. Once the date ranges are selected, a query string is created with the to and from dates. A key/value parameter input is created, where the key is the string "Query" and the value is a `GeoprocessingString` that contains the query string. These parameter inputs are then added to the `GeoprocessingParameters`. A `GeoprocessingJob` is obtained by calling the `createJob` method of the `GeoprocessingTask`. The job is started, and once it is complete, the `ArcGISMapImageLayer` is obtained from the result, and added to the `Map`.
-
-## Features
-- GeoprocessingTask
-- GeoprocessingJob
-- GeoprocessingParameters
-- GeoprocessingString
-- GeoprocessingResult
-- ArcGISMapImageLayer
-- Map
+
+1. Create a `GeoprocessingTask` with the URL set to the endpoint of a geoprocessing service.
+1. Create a query string with the date range as an input of `GeoprocessingParameters`.
```

Comment:
I'd suggest we use actual numbers instead of repeating `1.`. I know it works for rendering on github but I fear other processes or tools may not support this (the developers website or the sample viewer)

Reply:
Good catch! I'll update this in common design afterwards. I will keep a list of little stuff like this while I go through them all.